README.md

```diff
@@ -257,10 +262,14 @@ For documentation on how to locally test the Secrets Store CSI Driver Provider f
 
 ## Support
 
-Azure Key Vault Provider for Secrets Store CSI Driver is an open source project that is [**not** covered by the Microsoft Azure support policy](https://support.microsoft.com/en-us/help/2941892/support-for-linux-and-open-source-technology-in-azure). [Please search open issues here](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues), and if your issue isn't already represented please [open a new one](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/new/choose). The project maintainers will respond to the best of their abilities.  
+Azure Key Vault Provider for Secrets Store CSI Driver is an open source project that is [**not** covered by the Microsoft Azure support policy](https://support.microsoft.com/en-us/help/2941892/support-for-linux-and-open-source-technology-in-azure). [Please search open issues here](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues), and if your issue isn't already represented please [open a new one](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/new/choose). The project maintainers will respond to the best of their abilities.
 
 ## Presentations
 
 This demo created by <a href="https://twitter.com/HoussemDellai">Houssem Dellai</a> is using AAD Pod Identity and Secret Store CSI provider for Key Vault to retrieve database login and password from Azure Key Vault. <a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24">Watch it here.</a>
 
-<a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24"><img src="https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/images/demo-video-thumb.png?raw=true" width="60%"/></a>
+<a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24"><img src="https://github.com/HoussemDellai/secrets-store-csi-driver-provider-azure/blob/added-demo/images/demo-video-thumb.png?raw=true" width="60%"/></a>
```

Comment:
```
<a href="https://www.youtube.com/watch?v=dAFWrbeA6vQ&list=PLpbcUe4chE79sB7Jg7B4z3HytqUUEwcNE&index=24"><img src="https://github.com/HoussemDellai/secrets-store-csi-driver-provider-azure/blob/added-demo/images/demo-video-thumb.png?raw=true" width="60%"/></a>
```

@NileshGule Could you revert this change? The URL is fixed in the master and this change is adding back the old URL.

Reply:
ping @NileshGule to resolve the outstanding comment

Reply:
@aramase I have updated the link to the image. Please review the latest changes.